test.info

```diff
@@ -4,7 +4,7 @@ required
 {
   serviceName printer
   appPrefix /printer2/service-info
-  lifetime 100
+  lifetime 20
```

Comment:
Probably comment is required whether seconds, ms?
Also have a test.info.sample and don't modify that file in commits.

Reply:
done.